BUILDING.md

```diff
@@ -121,7 +121,7 @@ and not a newer version.
 
 To run the tests:
 
-```console
```

Comment:
I'm pretty sure I'm the one that added `console`. The idea was to distinguish between something that was a command (such as `make test`) and something which was console display (as it is here because it includes the `$` prompt).

All that said, I don't actually care if it goes away. :-D